packages/use-intl/src/core/MessageFormat.tsx

```diff
@@ -0,0 +1,9 @@
+import type {compile} from 'icu-to-json/compiler';
+
+type MessageFormat = Omit<
+  ReturnType<typeof compile>,
+  // TODO: Do we need the args?
+  'args'
```

Comment:
Could this be removed from the parsed result?

Reply:
use `compileToJson` instead - args are mainly for types or to omptimize which formatters should be included